slice/slice.go

```diff
@@ -700,12 +720,9 @@ func SortByField(slice interface{}, field string, sortType ...string) error {
 		}
 		a = a.FieldByIndex(sf.Index)
 		b = b.FieldByIndex(sf.Index)
-		return less(a, b)
+		return compare(a, b)
 	})
 
-	if sortType[0] == "desc" {
```

Comment:
fixes also a panic if sortType length  is  zero

@duke-git

Reply:
> fixes also a panic if sortType length is zero
> 
> @duke-git

@donutloop , great!